<?xml version="1.0"?>
<launch>
  <arg name="bag" value="/home/linh/Downloads/bags/convert_from_trajectory.bag"/>
  <node name="road_boundary" pkg="road_boundary" type="bag_process" >
    <!-- <rosparam file="$(arg file_path)" command="load" /> -->
    <param name="path" value="/home/linh/Downloads/bags/cleantech/grid_map_drivable.png" />
    <param name="cosine_threshold" value="0.9"/>
    <param name="distance_threshold" value="5.0"/>
    <param name="search_threshold" value="10.0"/>
    <param name="bag" value="$(arg bag)" />
    <param name="topic" value="/global_odom" />
    <param name="time_threshold" value="1.5"/>
    <param name="split_distance_threshold" value="70.0"/>
    <param name="min_set_size" value="5"/>
    <param name="tolerance" value="0.5"/>
    <!-- <remap from="road_boundary/topic" to="/global_topic" /> -->
  </node>
</launch>
